report/renderer/util.js

```diff
@@ -512,10 +512,12 @@ class Util {
         summary = cpuThrottling = networkThrottling = Util.i18n.strings.runtimeUnknown;
     }
 
-    const deviceEmulation = {
-      mobile: Util.i18n.strings.runtimeMobileEmulation,
-      desktop: Util.i18n.strings.runtimeDesktopEmulation,
-    }[settings.formFactor] || Util.i18n.strings.runtimeNoEmulation;
+    let deviceEmulation = Util.i18n.strings.runtimeMobileEmulation;
+    if (settings.screenEmulation.disabled) {
+      deviceEmulation = Util.i18n.strings.runtimeNoEmulation;
+    } else if (settings.formFactor === 'desktop') {
```

Comment:
I still think we should use `settings.screenEmulation.mobile` here because the strings refer to the emulated device and not scoring methodology.

That should only matter when the `formFactor` and `screenEmulation.mobile` disagree for some reason so this is probably fine.

Reply:
Good idea.